src/OrchardCore.Modules/OrchardCore.OpenId/Drivers/OpenIdClientSettingsDisplayDriver.cs

```diff
@@ -0,0 +1,94 @@
+using OrchardCore.OpenId.ViewModels;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.Extensions.Caching.Memory;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Notify;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Entities.DisplayManagement;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Settings;
+using System;
+using System.Threading.Tasks;
+using OrchardCore.OpenId.Settings;
+using OrchardCore.OpenId.Services;
+
+namespace OrchardCore.OpenId.Drivers
+{
+    public class OpenIdConnectSettingsDisplayDriver : SectionDisplayDriver<ISite, OpenIdClientSettings>
+    {
+        private const string RestartPendingCacheKey = "OpenIdConnect_RestartPending";
+        private const string SettingsGroupId = "OrchardCore.OpenId.Cient";
```

Comment:
Typo (that prevents the settings page from working correctly). I'll fix it for you.